<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>441</width>
    <height>437</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Sensor UI</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="sensorTab">
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <attribute name="title">
       <string>Sensors</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="SensorWidget" name="sensorWidget_1">
           <property name="identifier" stdset="0">
            <string>RW01</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="SensorWidget" name="sensorWidget_2">
           <property name="identifier" stdset="0">
            <string>RW02</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="SensorWidget" name="sensorWidget_4">
           <property name="identifier" stdset="0">
            <string>RW04</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="SensorWidget" name="sensorWidget_3">
           <property name="identifier" stdset="0">
            <string>RW03</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="WatchdogWidget" name="watchdogWidget"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="logTab">
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <attribute name="title">
       <string>Log</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="LogWidget" name="logWidget"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SensorWidget</class>
  </customwidget>
  <customwidget>
   <class>WatchdogWidget</class>
  </customwidget>
  <customwidget>
   <class>LogWidget</class>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
